2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,8 +23,13 @@
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
+  private final Hopper m_hopper = new Hopper();
+
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  XboxController driveController = new XboxController(Constants.kOI.DRIVE_CONTROLLER);
+  XboxController operatorController = new XboxController(Constants.kOI.OPERATOR_CONTROLLER);
```

Comment:
minor: please put access level on the variable
```suggestion
  private XboxController driveController = new XboxController(Constants.kOI.DRIVE_CONTROLLER);
  private XboxController operatorController = new XboxController(Constants.kOI.OPERATOR_CONTROLLER);
```